app/scripts/controllers/overview.js

```diff
@@ -80,11 +80,16 @@ function OverviewController($scope,
   var label = $filter('label');
   var getPodTemplate = $filter('podTemplate');
 
-  // API versions
+  var deploymentsVersion = APIService.getPreferredVersion('deployments');
+  var horizontalPodAutoscalersVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
   var serviceBindingsVersion = APIService.getPreferredVersion('servicebindings');
   var serviceClassesVersion = APIService.getPreferredVersion('clusterserviceclasses');
   var serviceInstancesVersion = APIService.getPreferredVersion('serviceinstances');
   var servicePlansVersion = APIService.getPreferredVersion('clusterserviceplans');
+  var statefulSetsVersion = APIService.getPreferredVersion('statefulsets');
+  var replicaSetsVersion = APIService.getPreferredVersion('replicasets');
+  $scope.buildConfigsInstantiateVersion = APIService.getPreferredVersion('buildconfigs/instantiate');
```

Comment:
This is a component, so we shouldn't be adding it to scope directly.

```
overview. buildConfigsInstantiateVersion = ...
```

Reply:
Ah, thats right, we have that pattern for overview.  updating, just a min.

Reply:
Yeah it uses controllerAs

https://github.com/openshift/origin-web-console/blob/master/app/scripts/app.js#L73-L78